### ISSUE TITLE
can you decrypt those？

### ISSUE THREAD

**Issue by llmmcc** [MiskaDaeve/AboutYsm#1]

**Comment by MiskaDaeve** [comment]
No, ysm 1.2.1 and the higher versions had written their loading logics into native libraries.And the native libraries was virtualized so that it's difficult to crack
